frame/example-offchain-worker/src/REMOVE_ME.rs

```diff
@@ -692,3 +692,19 @@ fn parse_price_works() {
 		assert_eq!(expected, Example::parse_price(json));
 	}
 }
+
+// Call contract in test.
+	Contracts::call(
+		Origin::signed(alice),
```

Comment:
Why do we need to change `REMOVE_ME` file?

Reply:
It’s just notes, we can remove this file.

Reply:
removed